src/org/thoughtcrime/securesms/ConversationTitleView.java

```diff
@@ -76,6 +76,21 @@ public void setTitle(@NonNull GlideRequests glideRequests, @Nullable Recipient r
     }
   }
 
+  public void setCustomeTitle(@NonNull GlideRequests glideRequests, @Nullable Recipient recipient,
```

Comment:
Not a big issue but I assume `Custome` is a typo.

Reply:
big enough to have to change it!